images/node-placeholder-scaler/scaler/scaler.py

```diff
@@ -137,6 +137,8 @@ def main():
                 ):
```

Comment:
see the line above (line 136):  i just double checked the logs and this conditional is NEVER met!  the actual output from the `kubectl` execution always returns `Setting XXXX to Y replicas`, even if the command doesn't change anything.
<img width="541" alt="Screenshot 2023-06-21 at 5 12 01 PM" src="https://github.com/berkeley-dsep-infra/datahub/assets/1606572/73f2bdcb-696e-435e-8370-560008820ec7">